app/tasks/raster_tile_cache_assets/utils.py

```diff
@@ -80,6 +90,7 @@ async def create_wm_tile_set_job(
     creation_options: RasterTileSetSourceCreationOptions,
     job_name: str,
     parents: Optional[List[Job]] = None,
+    use_resampler: bool = False,
```

Comment:
Do you need to pass this param through all these functions, or can you just determine whether to resample from the creation options here?

Reply:
Mmm, good question. Because we lose a little context it would slightly widen the range of circumstances in which the resampling script would run if for example we just checked "if creation_options.calc is None" in this function. I have to think carefully about what things might get included that it can't handle, or that at least haven't been tested. I keep changing my mind, but would like to err on the side of caution and save this for a future clean-up + testing. I'll add a TODO to that effect.